lib/application-processing/schema.ts

```diff
@@ -7,6 +7,7 @@ import { gql } from '@apollo/client';
 // TODO: Add nested resolver support for employees
 export default gql`
   type ApplicationProcessing {
+    id: Int!
```

Comment:
We shouldn't need an `id` field here, ID should not be queryable on an application processing object

Reply:
Added it initially because ts was mad in the invoice field resolver
<img width="919" alt="image" src="https://user-images.githubusercontent.com/55164714/160525131-f45a1d21-7977-4794-a6f3-581e6b6d2902.png">

But since it shouldn't be queryable, I just intersected an id type in the actual file. Lmk what u think